src/tokenizer/tokenizer.py

```diff
@@ -38,13 +38,14 @@
 
 """
 
+from typing import (Any, Callable, Deque, FrozenSet, Iterable, Iterator, List,
+                    Mapping, Match, Optional, Tuple, Type, TypeVar, Union,
```

Comment:
Use Black indentation style

Reply:
What is the max line length used in the project?